goreleaserlib/goreleaser.go

```diff
@@ -77,3 +81,26 @@ func Release(flags Flags) error {
 	log.Println("Done!")
 	return nil
 }
+
+// InitProject creates an example goreleaser.yml in the current directory
+func InitProject(filename string) error {
+	if _, err := os.Stat(filename); !os.IsNotExist(err) {
+		if err != nil {
+			return err
+		}
+
+		return fmt.Errorf("%s already exists", filename)
+	}
+
+	var ctx = context.New(config.Project{})
```

Comment:
I've noticed you prefer this style of declaration to `foo := bar` so I did the same here. But I'm curious to understand if it's just a taste preference or there's a technical reason for that.